include/deal.II/fe/mapping.h

```diff
@@ -640,7 +640,8 @@ class Mapping : public Subscriptor
    * The function above adjusted with the variable cell_normal_vectors for the
    * case of codimension 1
    */
-  virtual void
```

Comment:
This and the indentation will be fixed in the next patch that rewrites the whole interface, if you allow me to defer these issues.